### PR TITLE
267 tags on project list

### LINKED ISSUE #267: Add the tags to project cards shown on the theme, category, and our work pages
## Description

Projects on the work pages should contain a list of tags to show which categories & themes they belong to

(current)

Screenshot 2023-06-28 at 13 52 15

(proposed)

Screenshot 2023-06-28 at 13 51 15

https://www.figma.com/file/rLewb9kn4mUt8lLI0wuC7f/GFSC?type=design&node-id=245-314&mode=design&t=fQa9SLXXAEXhPLPT-0

## Acceptance criteria
- [ ] I can see a list of tags on a project card on the `our work` page
- [ ] I can click on a tag to visit the index page for that tag
- [ ] #268

### PR DESCRIPTION
**Pull request by aaaaargZombies**
Fixes #267 

## Description

- add themes to project preview cards
- link through to filtered theme list
- create a `themesTag` partial so we can add it to the project single page when we are ready
- use link styling because they are links!

@geeksforsocialchange/developers

### CODE REVIEW

**Comment by r-ferrier** [comment]
I like this but can't decide if they still look a bit close to the title to me - should the title and the body text maybe have even spacing from the tags, or is it nice to have them closer to the title than the text? @undividual do you have a preference? https://45863fef.gfsc.pages.dev/our-work/

**Review by r-ferrier — APPROVED** [review]
Looks good from a code perspective, have just asked for a second opinion on the spacing from Emma but approving because once that's confirmed think this will be good to go

**Comment by undividual** [comment]
I don't feel strongly — I like them as they are in that link you sent, so probably safest to leave as is! 3 or 4 more pixels of spacing MIGHT be nice but might also be... less nice! I'd need to see it and don't think it's worth the bother. So stick as is IMO.

**Pull request merged**
